src/main/java/com/moabam/api/application/room/mapper/ParticipantMapper.java

```diff
@@ -9,6 +11,13 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class ParticipantMapper {
 
+	public static Participant toParticipantEntity(Room room, Long memberId) {
```

Comment:
Q: 저는 걍 엔티티로 매핑하는 경우에는 Entity 안붙이긴 했는데 뭐가 더 나을까요?
```suggestion
	public static Participant toParticipant(Room room, Long memberId) {
```

Reply:
오잉 맞네 Entity는 빼는게 나을듯요!